pom.xml

```diff
@@ -420,6 +438,15 @@
                 source. This seemed to be the only solution -->
                 <configuration>
                     <argLine>--add-opens java.base/java.util=ALL-UNNAMED</argLine>
+                    <skipITs>${failsafe.skipITs}</skipITs>
+<!--                    <excludes>-->
+<!--                        <exclude>**/CaseActionEventIT.java</exclude>-->
+<!--                        <exclude>**/CaseReceiptReceiverIT.java</exclude>-->
+<!--                        <exclude>**/CaseActionEventIT.java</exclude>-->
+<!--                        <exclude>**/CaseEndpointIT.java</exclude>-->
+<!--                        <exclude>**/CaseIACEndpointIT.java</exclude>-->
+<!--                        <exclude>**/CachingTestIT.java</exclude>-->
+<!--                    </excludes>-->
```

Comment:
I'll remove these before merge. This was to allow me to run only one IT at a time during development